epic_manager/cli.py

```
@@ -511,19 +573,304 @@ def cleanup(config: Config, worktree_name: str) -> None:
+
+    Examples:
+        epic-mgr stack sync                    # Sync all worktrees
+        epic-mgr stack sync -i scottbot        # Sync specific instance
+        epic-mgr stack sync -e 355             # Sync specific epic
+        epic-mgr stack sync --auto             # No confirmation prompts
+        epic-mgr stack sync --dry-run          # Preview what would be synced
+    """
+    from pathlib import Path
+    import subprocess
+    from .workspace_manager import WorkspaceManager
+    from .graphite_integration import GraphiteManager
```

Ruff (0.13.3)

Redefinition of unused `GraphiteManager` from line 24

Remove definition: `GraphiteManager`

(F811)